environment/swarmball_env.py

```diff
@@ -4,17 +4,24 @@
 from .simulation.simulation import SwarmBallSimulation
 
 class SwarmBall(gym.Env):
-    def __init__(self):
-        self.sim = SwarmBallSimulation()
+    def __init__(self, acc_factor=0.25, thresh_amount=3, **kwargs):
+        self.sim = SwarmBallSimulation(**kwargs)
+        self.thresholds = [[] for _ in range(thresh_amount)]
+        self.thresh_vel = [0,0,0]
+        self.acc_factor = acc_factor
 
-    def seed(self, seed=None):
```

Comment:
Rozumiem ziarno wleci w innym miejscu nie jako osobna metoda?

Reply:
Na razie to usunąłem bo wydaje mi się że infrastruktura nie jest na to gotowa ze względu na ciągłe generowanie mapy na bieżąco. Nie wykluczam w przyszłości.

Reply:
Wrzuć to proszę to backlogu w takim razie